### DISCUSSION TITLE
Unable to load bundle identifiers

### DISCUSSION THREAD

**Discussion by emappstore777** [codemagic-ci-cd/codemagic-docs#1794] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hello! While setting up Flutter application for iOS - choosing code signing method "Automatic", choosing created API Key and receive an error "Unable to load bundle identifiers
Loading bundle identifiers for 1msonf failed. A required agreement is missing or has expired. - This request requires an in-effect agreement that has not been signed or has expired."

Please help why the error happens?

### Expected results

Hello! While setting up Flutter application for iOS - choosing code signing method "Automatic", choosing created API Key and receive an error "Unable to load bundle identifiers
Loading bundle identifiers for 1msonf failed. A required agreement is missing or has expired. - This request requires an in-effect agreement that has not been signed or has expired."

Please help why the error happens?

### Actual results

Hello! While setting up Flutter application for iOS - choosing code signing method "Automatic", choosing created API Key and receive an error "Unable to load bundle identifiers
Loading bundle identifiers for 1msonf failed. A required agreement is missing or has expired. - This request requires an in-effect agreement that has not been signed or has expired."

Please help why the error happens?

**Comment by hatefultester** [comment]
i can confirm that i have same issue with API key that is valid on appstore connect 

Unable to load bundle identifiers

Loading bundle identifiers for key failed. A required agreement is missing or has expired. - This request requires an in-effect agreement that has not been signed or has expired.

**Reply by himesh-cm** [reply, marked as answer]
Hi @emappstore777 @hatefultester 

Based on the error message, it appears that there are some agreements pending on your Apple Developer account. To resolve this issue, kindly visit https://appstoreconnect.apple.com/agreements/#/ and accept these agreements.

Thank you

**Reply by himesh-cm** [reply]
While I’m closing our thread for now, if you have any other questions or suggestions, feel free to reply back or start a new conversation anytime. Have a nice day!
